package.json

```diff
@@ -18,6 +18,7 @@
   },
   "dependencies": {
     "debug": "^3.1.0",
+    "lodash": "^4.17.10",
```

Comment:
perhaps `lodash` `^4.17.5` as the loosest version that doesn't have a vuln.. for some hope of de-duping

Reply:
https://snyk.io/vuln/npm:lodash

Reply:
:+1: